src/components/SelfHostedSection.tsx

```diff
@@ -14,19 +14,19 @@ const selfHostedFeatures: string[] = [
 ]
 
 export const SelfHostedSection: FunctionComponent = () => (
-    <div className="bg-gradient-venus py-5 py-md-7">
+    <div className="py-5 sg-bg-gradient-venus py-md-7">
         <div className="container">
-            <h1 className="font-weight-bold mb-6 text-center">Self-hosted deployment</h1>
+            <h2 className="mb-6 text-center">Self-hosted deployment</h2>
 
             <div className="row">
                 <div className="col-lg-6">
                     <Install />
                 </div>
 
                 <div className="col-lg-6 pl-lg-7">
-                    <h3 className="mt-6 mt-lg-0 font-weight-bold">Why choose self-hosted?</h3>
+                    <h3 className="mt-6 mt-lg-0">Why choose self-hosted?</h3>
```

Comment:
Leaving this one as is. I think we should always default to use what's defined from the DLS unless a design or requests states otherwise. But in that case, we should go back to design and enforce our DLS or change the DLS to suit the entire codebase. I think that's the goal. So I'm going to leave this one for the post-merge-audit.